### ISSUE TITLE
Search functionality bug in read mode

### ISSUE THREAD

**Issue by emalderson** [vnotex/vnote#2137]
**Describe the bug**
The plain search function in VNote treats terms separated by spaces as an OR instead of an AND in the Read mode.

**To Reproduce**
1. Ctrl+F
2. Uncheck all the search checkboxes
3. Search for "Hello World"
4. All the "Hello" words and the "World" words are highlighted

**Expected behavior**
Only the portions of text containing "Hello World" with the space in between the two words should be highlighted.

**Screenshots**
![image](https://user-images.githubusercontent.com/49370319/170106292-03559e9c-fade-4459-8588-821b0f1b7cee.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version: 3.13.0

**Comment by tamlok** [comment]
Will fix it in next version.

**Comment by emalderson** [comment]
Hi, regarding this bug I've tested the latest continuous integration build and I found out that the bug has been partially fixed. Indeed, if I search something containing a space that is placed in a code block, I receive two occurrences instead of one. For example, in this picture I searched for "type COMMAND" and despite the search result being correct, I see 2 occurrences.

![image](https://user-images.githubusercontent.com/49370319/187382354-8c6a6be0-a35c-4c01-8035-40ce94ef66b1.png)
